### PR TITLE
ICU-22922 ICU 76 final release

### PR DESCRIPTION
**Pull request by markusicu**
#### Checklist
- [x] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-22922
- [x] Required: The PR title must be prefixed with a JIRA Issue number. Example: "ICU-1234 Fix xyz"
- [x] Required: The PR description must include the link to the Jira Issue, for example by completing the URL in the first checklist item
- [x] Required: Each commit message must be prefixed with a JIRA Issue number. Example: "ICU-1234 Fix xyz"
- [x] Issue accepted (done by Technical Committee after discussion)
- [ ] Tests included, if applicable
- [x] API docs and/or User Guide docs changed or added, if applicable

### CODE REVIEW

File: docs/download/76.md [hunk 1]
```diff
@@ -242,17 +242,16 @@ ICU4J should work on Android API level 21 and later but may require “[library
 
 ## Download
 
-Source and binary downloads are available on the git/GitHub tag page: https://github.com/unicode-org/icu/releases/tag/release-76-rc
+Source and binary downloads are available on the git/GitHub tag page: <https://github.com/unicode-org/icu/releases/tag/release-76-rc>
```
Comment (mihnita) [inline docs/download/76.md:245]:
```suggestion
Source and binary downloads are available on the git/GitHub tag page: <https://github.com/unicode-org/icu/releases/tag/release-76>
```

File: docs/download/76.md [hunk 2]
```diff
@@ -242,17 +242,16 @@ ICU4J should work on Android API level 21 and later but may require “[library
...
 
 See the [Source Code Setup](../devsetup/source/) page for how to download the ICU file tree directly from GitHub.
 
 ICU locale data was generated from CLDR data equivalent to:
 
-* https://github.com/unicode-org/cldr/releases/tag/release-46-beta3
-* https://github.com/unicode-org/cldr-staging/releases/tag/release-46-beta3
+* <https://github.com/unicode-org/cldr/releases/tag/release-46-beta3>
```
Comment (mihnita) [inline docs/download/76.md:251]:
```suggestion
* <https://github.com/unicode-org/cldr/releases/tag/release-46>
```

File: docs/download/76.md [hunk 3]
```diff
@@ -242,17 +242,16 @@ ICU4J should work on Android API level 21 and later but may require “[library
...
+* <https://github.com/unicode-org/cldr-staging/releases/tag/release-46-beta3>
```
Comment (mihnita) [inline docs/download/76.md:252]:
```suggestion
* <https://github.com/unicode-org/cldr-staging/releases/tag/release-46>
```

**Review by mihnita — COMMENTED** [review]
Fixed URLs to remove `rc` and `beta3`

File: docs/download/76.md (hunk 1 above)
Comment (markusicu) [reply docs/download/76.md:245]: actually, with an additional -1 [as usual](https://github.com/unicode-org/icu/releases/tag/release-75-1)

done tnx

File: docs/download/76.md (hunk 2 above)
Comment (markusicu) [reply docs/download/76.md:251]: ok -- I got confirmation that for ICU beta3 is really what we have, but the anticipated final-release URL looks cleaner

done

**Review by mihnita — DISMISSED** [review]

**Review by richgillam — APPROVED** [review]
LGTM

File: docs/download/index.md [hunk 4]
```diff
@@ -15,15 +15,11 @@ License & terms of use: http://www.unicode.org/copyright.html
 If you want to use ICU (as opposed to developing it), it is recommended that you download an official packaged version of the ICU source code. These versions are tested more thoroughly than day-to-day development builds of the system, and they are packaged in zip and tar files for convenient download. Here are several recent releases of ICU that are available:
 
 ## Release Candidate
```
Comment (mihnita) [inline docs/download/index.md:17]: Oups
"Release"
?

File: docs/download/index.md [hunk 5]
```diff
@@ -15,15 +15,11 @@ License & terms of use: http://www.unicode.org/copyright.html
...
 
 Most of the java.time (Temporal) types can now be formatted directly. There are some new APIs to make ICU easier to use with modern C++ and Java patterns. The Java and C++ technology preview implementations of the CLDR MessageFormat 2.0 specification have been updated to match recent changes. See [ICU 76](76.md).
 
-Please test this release candidate on your platforms and report bugs and regressions by Monday, 2024-oct-21, via the [icu-support](https://icu.unicode.org/contacts) mailing list, and/or please [find/submit error reports](https://icu.unicode.org/bugs).
-
-Please do not use this release candidate in production.
-
 ## Latest Release
```
Comment (mihnita) [inline docs/download/index.md:23]: "## Previous Release"
?

**Comment by yumaoka** [comment]
@markusicu Can you add a line for ICU-22596?

Updated IBM EBCDIC code page 1388 (Simplified Chinese) mapping data to ibm-cp1388_P100-2024 (See ICU-22596)

File: docs/download/index.md (hunk 4 above)
Comment (echeran) [reply docs/download/index.md:17]: This page is the landing page for ICU downloading in general. Once the final release is made, do we remove the "Release Canadidate" section until the next version's RC is created?

File: docs/download/index.md (hunk 5 above)
Comment (echeran) [reply docs/download/index.md:23]: When people go to the general ICU download page, we should always point them to the latest release.

Maybe that means that the description for the ICU 76 release candidate moves into the Latest Release section, and the description for ICU 75 moves into the Previous Releases section?

**Comment by mihnita** [comment]
The issue number in this PR is ICU-22922
That is "ICU 77rc BRS"

Should it be [ICU-22724, "ICU 76 BRS"](https://unicode-org.atlassian.net/browse/ICU-22724)?

**Comment by markusicu** [comment]
> The issue number in this PR is [ICU-22922](https://unicode-org.atlassian.net/browse/ICU-22922) That is "ICU 77rc BRS"
> 
> Should it be [ICU-22724, "ICU 76 BRS"](https://unicode-org.atlassian.net/browse/ICU-22724)?

No, because docs need to be edited on the main branch, and that's past 76 now.

**Comment by markusicu** [comment]
> @markusicu Can you add a line for [ICU-22596](https://unicode-org.atlassian.net/browse/ICU-22596)?

done

**Review by echeran — APPROVED** [review]

**Pull request merged**